bpf/lib/overloadable_skb.h

```diff
@@ -183,6 +183,30 @@ static __always_inline bool ctx_snat_done(const struct __sk_buff *ctx)
 	return (ctx->mark & MARK_MAGIC_HOST_MASK) == MARK_MAGIC_SNAT_DONE;
 }
 
+static __always_inline __u32 get_tunnel_id(__u32 identity)
```

Comment:
Why do we need to define these functions twice? There is nothing specific to skb or xdp buffers here. We should be able to define these functions once in `lib/identity.h`?

Reply:
Ah right. I guess `lib/common.h` is an option to avoid duplication.